package.json

```diff
@@ -20,5 +20,9 @@
     "gulp-postcss": "^4.0.3",
     "gulp-sass": "^1.3.3",
     "gulp-sourcemaps": "^1.5.0"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "gulp"
```

Comment:
I was a little unsure on the naming.  We could name it "build" instead (eg `npm run build`), but "start" is the conventional entrypoint to run a module (eg `npm start`).  You would only ever run this to rebuild it, but running and building are conceptually different.  I will rename it to `build`.